Section 1/Task1/style.css

```diff
@@ -61,4 +64,103 @@
     top: 55px;
     left: 4px;
     position: absolute;
+}
+
+@media screen and (max-width: 800px) {
+    body{
+        background-color: rgb(52, 130, 37);
+    }
+
+    .container{
+        width: 450px;
+        height: 450px;
+        border: 8px solid #fff;
+        border-style: ridge;
+        margin: 55px auto;
+    }
+    
+    .triangle{
+        border-left: 210px solid blue;
+        border-right: 210px solid blue;
+        border-bottom: 130px solid #d7d07b;
+        margin: 65px auto;
+    }
+    
+    .circle{
+        width: 60px;
+        height: 60px;
+        top: 35px;
+        left: -31px;
+    }
+    
+    .rectangle{
+        height: 180px;
+        width: 300px;
+        margin: -65px auto;
+    }
+    
+    .rectangle1{
+        width: 85px;
+        height: 101px;
+        top: 79px;
+        left: 107px;
+    }
+    
+    .circle1{
+        width: 10px;
+        height: 10px;
+        top: 49px;
+        left: 3px;
+    } 
+}
+
+@media screen and (max-width: 480px) {
+    body{
+        background-color: rgb(164, 4, 4);
+    }
+
+    .container{
+        width: 350px;
+        height: 350px;
+        border: 6px solid #fff;
+        border-style: ridge;
+        margin: 55px auto;
+    }
+
+    .triangle
+        {
+            border-left: 165px solid blue;
+            border-right: 165px solid blue;
+            border-bottom: 110px solid #d7d07b;
+            margin: 32px auto;
+        }
+    
+    
+    .circle{
+        width: 45px;
+        height: 45px;
+        top: 36px;
+        left: -24px;
+    }
+    
+    .rectangle{
+        height: 150px;
+        width: 244px;
+        margin: -32px auto;
+    }
+    
+    .rectangle1{
```

Comment:
change names of class...don't use names like circle1, rectangle1, etc.

Reply:
Okay

Reply:
Done. Kindly please let me know if its fine or any other changes that needs to done.